pkgs/tools/security/fwbuilder/default.nix

```diff
@@ -1,4 +1,13 @@
-{ stdenv, lib, fetchFromGitHub, cmake, qtbase, wrapQtAppsHook }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qtbase
+, wrapQtAppsHook
+, wayland
+, wayland-protocols
+, qtwayland
```

Comment:
```suggestion
, qtwayland
```

Reply:
Thats used in line 31

Reply:
@Artturin after removing that line the build fails:

```
$ head ./pkgs/tools/security/fwbuilder/default.nix
{ lib
, stdenv
, fetchFromGitHub
, cmake
, qtbase
, wrapQtAppsHook
, wayland
, wayland-protocols
}:
```

With this error:

```
$ nix-build $NIXPKGS -A fwbuilder
error: undefined variable 'qtwayland' at /data/nixpkgs/pkgs/tools/security/fwbuilder/default.nix:30:5
```